src/styles/blocks/product.scss

```diff
@@ -0,0 +1,80 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  width: $product-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__disc {
+    font-weight: $font-weight-med;
+    font-size: $primary-sm-font;
+    padding: 40px 0 4px;
+    line-height: 18px;
+    margin: 0;
+    color: $text-primary-color;
+  }
+
+  &__code {
+    align-self: flex-start;
+    font-weight: 400;
+    font-size: $secondary-extra-sm-font;
+    line-height: 14px;
+    color: $text-secondary-color;
+    margin: 0 0 16px;
+  }
+
+  &__price {
+    display: flex;
+    align-self: flex-start;
+    margin-top: 24px;
+    justify-content: space-between;
+    line-height: 18px;
+    align-items: center;
+    width: 100%;
+  }
+
+  &__text {
+    font-size: $primary-sm-font;
+    font-weight: 400;
+    margin: 0;
+    color: $text-secondary-color;
+  }
+
+  &__number {
+    font-size: 16px;
+    font-weight: $font-weight-bold;
+    margin: 0;
+    color: $text-primary-color;
+  }
+
+  &__buy {
+    margin-top: 16px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    background-color: $primary-style-color;
+    color: #fff;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    text-decoration: none;
+    font-size: 14px;
+    line-height: 16px;
+    padding: 12px 70px;
+    font-weight: $font-weight-bold;
+
+    &:hover {
```

Comment:
Maybe add transitioning here?

Reply:
add transition property on hover?

Reply:
You add it for the properties that are changed on hover, but `transition` rule is added under general selector.